linux/LinuxProcessList.c

```diff
@@ -1989,9 +2004,7 @@ void ProcessList_goThroughEntries(ProcessList* super, bool pauseProcessUpdate) {
       return;
    }
 
-   struct timeval tv;
-   gettimeofday(&tv, NULL);
-   unsigned long long now = tv.tv_sec * 1000ULL + tv.tv_usec / 1000ULL;
+   unsigned long long now = super->timestamp.tv_sec * 1000ULL + super->timestamp.tv_usec / 1000ULL;
```

Comment:
Given that it's repeated quite a few times, a prerequisite patch adding a `tv_to_msec()` might be useful.

Reply:
It exists in two different places, each one of them has its own calculation, the rest of the duplicate places are removed.

Reply:
I can't follow. I still see it here and in `LinuxProcessList_updateTime`.

Reply:
Can't `now` be replaced by `pl->timestampMs`?

Reply:
If `unsigned long long now` == `uint64_t pl->timestampMs`? Yes :) (and I 'think' it-is)